docs/generate_svgs.js

```diff
@@ -0,0 +1,49 @@
+const fs = require("fs");
```

Comment:
1. Add some documentation explaining the purpose of this script.
2. Add a shebang line: `#!/usr/bin/env node`. Change the permission to make this script executable (git can track permissions as well).
3. Move this file to a new `docs/scripts` subdirectory (in general, avoid putting helper scripts in top-level directories, since that can create clutter).